packs/data/pathfinder-bestiary-3.db/adult-sovereign-dragon-spellcaster.json

```diff
@@ -2703,7 +2703,7 @@
                 }
             },
             "img": "systems/pf2e/icons/spells/detect-alignment.webp",
-            "name": "Detect Alignment (At Will) (Good or Evil Only)",
+            "name": "Detect Alignment (At Will) ((Evil or Good Only))",
```

Comment:
```suggestion
            "name": "Detect Alignment (At Will) ((Evil or Good Only))",
```
Ended up with extra parenthesis. Also looks like the books uses `Good or Evil` as the order.

![image](https://user-images.githubusercontent.com/80183198/205824883-ab41984f-7894-4edc-ab63-34a1be7305d0.png)

I'd like to stick to Paizo's style, and if they have a conflicting style then I'd personally prefer to be "Good or Evil" instead of the other way around, since many English idioms use that ordering sop it sounds more natural than going in alphabetical order.

Reply:
I agree